web/angular-wallet/src/app/components/page/page.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="page-layout blank simple" fusePerfectScrollbar>
```

Comment:
potential overabstraction IMO

Reply:
you mean the page component itself? If yes, I simply disagree :grin: